tasks/main.yml

```diff
@@ -1,9 +1,4 @@
 ---
-- include_vars: "{{ item }}"
-  with_first_found:
-    - "../vars/{{ ansible_os_family }}.yml"
```

Comment:
There's a reason why I did it this way; there are separate variables for Debian and Redhat.